src/web/client/webViews/QuickPickProvider.ts

```diff
@@ -44,11 +51,55 @@ export class QuickPickProvider {
         } else {
             this.items = [{
                 label: Constants.WEB_EXTENSION_QUICK_PICK_DEFAULT_STRING,
+                id: "",
             }];
         }
     }
 
-    public showQuickPick() {
-        vscode.window.showQuickPick(this.items);
+    public async showQuickPick() {
```

Comment:
nit: can we rename this to selectUserWithQuickPick()? showQuickPick() is native method of VSCode

Reply:
the method was only invoking that showQuickPick method of vscode, thats why gave same name